src/description.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var s = require('util').format;
+var marked = require('marked');
```

Comment:
> Then the only proper solution IMO is to have description and descriptionFile keys

That would be a good solution.
But sill, a user could pass in 'whatever.md' and not put markdown in it... ?
So is this really solving your concerns ?

> or to make a configurable filePattern.

Example ?

> Really strange idea, but then it would not work.

Does not feel so strange to me. I see it more as granularity. Maybe a custom theme want to use the raw annotations but still benefit from a Markdown description ?